src/main/java/bio/terra/common/iam/SamUserFactory.java

```diff
@@ -28,16 +29,16 @@
 @Component
 public class SamUserFactory {
   final BearerTokenFactory bearerTokenFactory;
-  final OkHttpClient httpClient =
-      new ApiClient()
-          .getHttpClient()
-          .newBuilder()
-          .addInterceptor(new OkHttpClientTracingInterceptor(Tracing.getTracer()))
-          .build();
+  final OkHttpClient httpClient;
 
-  @Autowired
-  public SamUserFactory(BearerTokenFactory bearerTokenFactory) {
+  @Autowired(required = false)
+  public SamUserFactory(
+      BearerTokenFactory bearerTokenFactory, Optional<OpenTelemetry> openTelemetry) {
     this.bearerTokenFactory = bearerTokenFactory;
+    var apiClientBuilder = new ApiClient().getHttpClient().newBuilder();
+    openTelemetry.ifPresent(
```

Comment:
When do we expect that openTelemtry won't be present?

Reply:
this is a library, I was trying not to force anything